aerofiles/seeyou/reader.py

```diff
@@ -193,3 +193,90 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_taskname(self, fields):
+        if not fields:
+            raise ParserError('Name field must not be empty')
+
+        return fields[0]
+
+    def decode_tasklist(self, fields, waypoints):
```

Comment:
thanks for the elaborate comment, this seems like a good plan.

i am not really experienced with branching and neat version control (as is already getting apparent in this messy PR xp). what is the best way to continue?